fuzzing/file_fuzzer.cc

```diff
@@ -85,8 +85,11 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
   struct heif_image_handle* primary_handle = nullptr;
   int images_count;
   heif_item_id* image_IDs = NULL;
+  bool explicit_init = data[size - 1] & 1;
```

Comment:
Seems we need an `size ? (data[size - 1] & 1) : false`

Reply:
Indeed, I committed 4ae47b2c3f88f1b86a561cfa4bc5b60018b21063 before noticing this review comment. Let me know if you want the else-clause (`size == 0`) to be `false` instead.

Reply:
I guess it doesn't matter.